src/Router/fetchRootComponent.js

```diff
@@ -11,17 +11,13 @@ export default function fetchRootComponent(chunkID, moduleID) {
         const modNamespace = webpackInterop.require(moduleID);
         if (!modNamespace) {
             throw new Error(
-                `Expected chunkID ${chunkID} to have module ${
-                    moduleID
-                }. Cannot render this route without a matching RootComponent`
+                `Expected chunkID ${chunkID} to have module ${moduleID}. Cannot render this route without a matching RootComponent`
```

Comment:
I don't see any updates to prettier's version in `package.json` - what changed here?

Reply:
It's possible that prettier didn't run on this file before, or ran with different settings; in any case I'll pull it out.